impeller/entity/contents/filters/matrix_filter_contents.cc

```diff
@@ -28,6 +28,17 @@ void MatrixFilterContents::SetSamplerDescriptor(SamplerDescriptor desc) {
   sampler_descriptor_ = std::move(desc);
 }
 
+namespace {
+Matrix CalculateSubpassTransform(const Matrix& entity_transform,
```

Comment:
@bdero this is what I was talking about the other day.  We should be sharing the math between rendering and coverage.